WalletLibrary/WalletLibrary/Protocols/Identifier/VerifiedIdIdentifier.swift

```diff
@@ -0,0 +1,26 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+/// An object that describes the holder of Verified IDs.
+protocol HolderIdentifier
+{
+    /// The unique identifier (ex. did:web:microsoft.com).
+    var id: String { get }
+
+    /// The algorithm used for cryptographic operations, such as signing.
```

Comment:
for clarity, could you specify [JWA "alg"](https://datatracker.ietf.org/doc/html/rfc7518#section-3.1) (RFC 7518 3.1, 4.1)